fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -349,6 +350,10 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     public static final String GROUP_CONCAT_MAX_LEN = "group_concat_max_len";
 
+    public static final String FULL_SORT_MAX_BUFFERED_ROWS = "full_sort_max_buffered_rows";
+
+    public static final String FULL_SORT_MAX_BUFFERED_BYTES = "full_sort_max_buffered_bytes";
```

Comment:
Add some comments in `variables.md`?

Reply:
These two variables are not easy for user to set.

Maybe be should automatically handle this based on the memory usage later.

Reply:
these two variables are turned into invisible.